code/modules/unit_tests/create_and_destroy.dm

```diff
@@ -98,6 +98,7 @@ GLOBAL_VAR_INIT(running_create_and_destroy, FALSE)
 	ignore += typesof(/obj/machinery/computer/holodeck)
 	//runtimes if not paired with a landmark
 	ignore += typesof(/obj/structure/industrial_lift)
+	ignore += create_and_destroy_voidcrew_ignores() // VOIDCREW ADDITION
```

Comment:
I think it would be better to make it not cry on init instead, we want to avoid tg edits